### PR TITLE
[CARBONDATA-3742] Support spark 2.4.5 integration

### PR DESCRIPTION
**Pull request by jackylk**
### Why is this PR needed?
Currently CarbonData does not support integration with spark 2.4.5
 
 ### What changes were proposed in this PR?
Support integration with spark 2.4.5
    
 ### Does this PR introduce any user interface change?
 - No

 ### Is any new testcase added?
 - No

### CODE REVIEW

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.4.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.4/778/

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2487/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.4/781/

**Comment by jackylk** [comment]
retest this please

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2496/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.4/791/

File: docs/prestodb-guide.md [hunk 1]
```diff
@@ -238,10 +238,10 @@ Now you can use the Presto CLI on the coordinator to query data sources in the c
   ```
   Replace the spark and hadoop version with the version used in your cluster.
   For example, use prestodb profile and 
-  if you are using Spark 2.4.4, you would like to compile using:
```
Comment (ajantha-bhat) [inline docs/prestodb-guide.md]: prestosql-guide also has this.

please look up 2.4.4

File: integration/spark/src/main/scala/org/apache/spark/util/CarbonReflectionUtils.scala [hunk 2]
```diff
@@ -31,7 +31,7 @@ import org.apache.spark.sql.catalyst.expressions.{Attribute, AttributeReference,
 import org.apache.spark.sql.catalyst.parser.AstBuilder
```
Comment (ajantha-bhat) [inline integration/spark/src/main/scala/org/apache/spark/util/CarbonReflectionUtils.scala]: unused import ?

**Comment by ajantha-bhat** [comment]
Once it is merged, we can rename the PR builder also to Spark 2.4.5

File: docs/prestodb-guide.md (hunk 1 above)
Comment (jackylk) [reply docs/prestodb-guide.md]: fixed

File: integration/spark/src/main/scala/org/apache/spark/util/CarbonReflectionUtils.scala (hunk 2 above)
Comment (jackylk) [reply integration/spark/src/main/scala/org/apache/spark/util/CarbonReflectionUtils.scala]: removed

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2526/

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.4.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.4/819/

**Comment by ajantha-bhat** [comment]
retest this please

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2535/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.4/828/

**Comment by ajantha-bhat** [comment]
LGTM

**Pull request closed without merging**